dashboards/consul/consul-prometheus.json

```diff
@@ -35,22 +35,25 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "consul_kvs_apply{${Cluster},${Location},${Namespace}}"
+                  "prometheusQuery": "consul_kvs_apply{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
                 }
               }
             ],
+            "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
-          }
-        },
-        "width": 6,
-        "xPos": 0,
-        "yPos": 6
+          },
+          "computedId": "ad6263be-e3bc-4945-9d17-86e7d0cadc3b"
```

Comment:
Please remove all instances of this `computedId` field, this appears to be specific to one instance of the dashboard.